_config.yml

```diff
@@ -55,6 +55,51 @@ FLINK_GITHUB_REPO_NAME: flink-statefun
 #          md1_url: https://repo.maven.apache.org/maven2/org/apache/flink/flink-metrics-prometheus_2.12/1.7.1/flink-metrics-prometheus_2.12-1.7.1.jar.sha1
 
 flink_releases:
+  -
+      version_short: "1.11"
+      binary_release:
+          name: "Apache Flink 1.11.0"
+          scala_211:
+              id: "1110-download_211"
+              url: "https://www.apache.org/dyn/closer.lua/flink/flink-1.11.0/flink-1.11.0-bin-scala_2.11.tgz"
+              asc_url: "https://downloads.apache.org/flink/flink-1.11.0/flink-1.11.0-bin-scala_2.11.tgz.asc"
+              sha512_url: "https://downloads.apache.org/flink/flink-1.11.0/flink-1.11.0-bin-scala_2.11.tgz.sha512"
+          scala_212:
+              id: "1110-download_212"
+              url: "https://www.apache.org/dyn/closer.lua/flink/flink-1.11.0/flink-1.11.0-bin-scala_2.12.tgz"
+              asc_url: "https://downloads.apache.org/flink/flink-1.11.0/flink-1.11.0-bin-scala_2.12.tgz.asc"
+              sha512_url: "https://downloads.apache.org/flink/flink-1.11.0/flink-1.11.0-bin-scala_2.12.tgz.sha512"
+      source_release:
+          name: "Apache Flink 1.11.0"
+          id: "1110-download-source"
+          url: "https://www.apache.org/dyn/closer.lua/flink/flink-1.11.0/flink-1.11.0-src.tgz"
+          asc_url: "https://downloads.apache.org/flink/flink-1.11.0/flink-1.11.0-src.tgz.asc"
+          sha512_url: "https://downloads.apache.org/flink/flink-1.11.0/flink-1.11.0-src.tgz.sha512"
+      optional_components:
+        -
+          name: "Avro SQL Format"
+          category: "SQL Formats"
+          scala_dependent: false
+          id: 1110-sql-format-avro
+          url: https://repo.maven.apache.org/maven2/org/apache/flink/flink-avro/1.11.0/flink-avro-1.11.0.jar
+          asc_url: https://repo.maven.apache.org/maven2/org/apache/flink/flink-avro/1.11.0/flink-avro-1.11.0.jar.asc
+          sha_url: https://repo.maven.apache.org/maven2/org/apache/flink/flink-avro/1.11.0/flink-avro-1.11.0.jar.sha1
+        -
+          name: "CSV SQL Format"
+          category: "SQL Formats"
+          scala_dependent: false
+          id: 1110-sql-format-csv
+          url: https://repo.maven.apache.org/maven2/org/apache/flink/flink-csv/1.11.0/flink-csv-1.11.0.jar
+          asc_url: https://repo.maven.apache.org/maven2/org/apache/flink/flink-csv/1.11.0/flink-csv-1.11.0.jar.asc
```

Comment:
json/csv formats are now bundled in lib/, so a separate download seems unnecessary.

Reply:
Shouldn't this be mentioned in the release notes?

Reply:
Yes. It appears this was missed in https://issues.apache.org/jira/browse/FLINK-18173.

Reply:
I've removed those links and I'm adding release notes